packages/http-client-python/generator/pygen/codegen/models/operation_group.py

```diff
@@ -152,6 +152,27 @@ def imports(self, async_mode: bool, **kwargs: Any) -> FileImport:
                 f"{self.client.name}MixinABC",
                 ImportType.LOCAL,
             )
+        else:
+            file_import.add_submodule_import(
+                "" if self.code_model.is_azure_flavor else "runtime",
+                f"{'Async' if async_mode else ''}PipelineClient",
+                ImportType.SDKCORE,
+            )
+            file_import.add_submodule_import(
+                ".._configuration",
```

Comment:
We shall always use `get_relative_import_path` to calculate relative import path otherwise it may fail when typespec namespace is enabled.